docroot/modules/custom/webform_serialization/tests/src/Functional/WebformNormalizationFormOptionsTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\Tests\webform_serialization\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\webform\Entity\Webform;
+use Drupal\Component\Serialization\Json;
+use Drupal\webform\Entity\WebformOptions;
+
+/**
+ * Tests for webform select options.
+ *
+ * @group foiaapi
+ */
+class WebformNormalizationFormOptionsTest extends BrowserTestBase {
```

Comment:
Typically, the test class name is the same as the class you're testing, with the word "Test" tacked onto the end. So in this case: `WebformNormalizerTest`.